spring-cloud-dataflow-docs/src/main/asciidoc/configuration-database.adoc

```diff
@@ -0,0 +1,65 @@
+
+[[configuration-database-overview]]
+A relational database is used to store stream and task definitions as well as the state of executed tasks.
+Spring Cloud Data Flow provides schemas for *MariaDB*, *MySQL*, *Oracle*, *PostgreSQL*, *Db2*, *SQL Server*, and *H2*. The schema is automatically created when the server starts.
+
+NOTE: The JDBC drivers for *MariaDB*, *MySQL* (via the _MariaDB_ driver), *PostgreSQL*, *SQL Server* are available without additional configuration. To use any other database you need to put the corresponding JDBC driver jar on the classpath of the server as described <<#add-custom-driver,here>>.
+
+To configure a database the following properties must be set:
+
+* `spring.datasource.url`
+* `spring.datasource.username`
+* `spring.datasource.password`
+* `spring.datasource.driver-class-name`
```

Comment:
should we show a sample, just one, as in the case of the code snippits that were removed below showing `java -jar sdcdf.jar --spring.datasource.url` or is that overkill?

Reply:
It is still there, but below this section.